pkg/store/bucket.go

```diff
@@ -402,11 +402,8 @@ func (s *BucketStore) blockSeries(ctx context.Context, b *bucketBlock, matchers
 	level.Debug(s.logger).Log("msg", "preload postings", "duration", time.Since(begin))
 
 	begin = time.Now()
-	var ps []uint64
-	for lazyPostings.Next() {
-		ps = append(ps, lazyPostings.At())
-	}
-	if err := lazyPostings.Err(); err != nil {
+	ps, err := index.ExpandPostings(lazyPostings)
```

Comment:
this is doing exactly the same logic?

Reply:
Yes, just exported it upstream recently when moving to a separate index package.

Reply:
nice